src/incubator/Dialog/DialogHeader.tsx

```diff
@@ -21,7 +22,24 @@ const DialogHeader = (props: DialogHeaderProps = {}) => {
       return renderContent(props);
     }
 
-    return <HeaderContent text={text}/>;
+    if (title || subtitle) {
```

Comment:
Why boolean comparison here is simple, but below it uses `isEmpty`/.

Reply:
Good point